### PR TITLE
6.1.2 release notes

### PR DESCRIPTION
**Pull request by Chr1st0ph3rTurn3r**

### CODE REVIEW

File: docs/release_notes_128t_6.1.md [hunk 1]
```diff
@@ -49,6 +49,75 @@ This issue will be corrected in an upcoming release.
...
+------
+- **I95-51006 Nodes stuck in connected state after upgrade:** On an HA conductor, if the user is performing an upgrade on the first conductor node and that user makes a config commit during the upgrade, then the configuration's modified time will become out of sync between the two conductor nodes. When the conductor first node is finished upgrading the result is a loop where the configuration keeps getting committed by each node back and forth until a new commit is made. This issue has been resolved by allowing the peer conductor node to accept the config despite the perceived version disparity. Please note performing a commit mid upgrade is not supported. 
+------
+- **I95-51007 cpusetOrchestrator isolating and pinning cores on the Conductor:** The cpuProperties cores setting in /etc/128technology/local.init was erroneously isolating cores on conductor nodes when set, even though this setting is intended for a router. This would cause a reduction in available processing cores for normal conductor operations. This setting will now be ignored on the conductor.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.1.md:111]: heading should be: "Conductor is incorrectly honoring core pinning"

File: docs/release_notes_128t_6.1.md [hunk 2]
```diff
@@ -49,6 +49,75 @@ This issue will be corrected in an upcoming release.
...
+------
+- **I95-51021 Package to Image conversion fails on FIPS enabled SSR:** Conversion of package-based to image-based is now supported for systems with FIPS 140-2 mode enabled.
+------
+- **I95-51044 Hide forwarding-core-mode on conductor:** Disabled the forwarding-core-mode setting on conductor nodes, since this setting doesn't apply to conductor.
+------
+- **I95-51086 Highway Crash on Headend:** Resolved an issue where a routing change that affects the `forwarding-table` can incur a race condition with sessions completing and being removed, which could lead to highway crashing and restarting.
+------
+- **I95-51093 Segfault in Highway when using multithreaded ServiceArea:** Resolved an issue when `session-scaling` is set to `enabled` together with `outbound-only`, whereby a race condition can cause a crash and restart of the highway process in the SSR.
+
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.1.md:120]: - [ ] @migolnikov has indicated that we need to add a caveat for the conductor proxy issue.

File: docs/release_notes_128t_6.1.md [hunk 3]
```diff
@@ -49,6 +49,75 @@ This issue will be corrected in an upcoming release.
...
+------
+- **I95-51021 Package to Image conversion fails on FIPS enabled SSR:** Conversion of package-based to image-based is now supported for systems with FIPS 140-2 mode enabled.
+------
+- **I95-51044 Hide forwarding-core-mode on conductor:** Disabled the forwarding-core-mode setting on conductor nodes, since this setting doesn't apply to conductor.
+------
+- **I95-51086 Highway Crash on Headend:** Resolved an issue where a routing change that affects the `forwarding-table` can incur a race condition with sessions completing and being removed, which could lead to highway crashing and restarting.
+------
+- **I95-51093 Segfault in Highway when using multithreaded ServiceArea:** Resolved an issue when `session-scaling` is set to `enabled` together with `outbound-only`, whereby a race condition can cause a crash and restart of the highway process in the SSR.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.1.md:119]: multi-threaded ServiceArea

**Review by MichaelBaj — CHANGES_REQUESTED** [review]

**Review by migolnikov — DISMISSED** [review]

**Review by MichaelBaj — CHANGES_REQUESTED** [review]

**Review by MichaelBaj — APPROVED** [review]

**Review by MichaelBaj — APPROVED** [review]

**Review by MichaelBaj — DISMISSED** [review]

File: docs/release_notes_128t_6.1.md [hunk 4]
```diff
@@ -49,6 +49,81 @@ This issue will be corrected in an upcoming release.
...
+------
+- **I95-51007 Conductor is incorrectly honoring core pinning:** The cpuProperties cores setting in /etc/128technology/local.init was erroneously isolating cores on conductor nodes when set, even though this setting is intended for a router. This would cause a reduction in available processing cores for normal conductor operations. This setting will now be ignored on the conductor.
+------
+- **I95-51021 Package to Image conversion fails on FIPS enabled SSR:** Conversion of `package-based` to `image-based` is now supported for systems with FIPS 140-2 mode enabled.
+------
+- **I95-51044 Hide `forwarding-core-mode` on conductor:** Disabled the `forwarding-core-mode` setting on conductor nodes, since this setting doesn't apply to conductor.
+------
+- **I95-51086 Highway Crash on Headend:** Resolved an issue where a routing change that affects the `forwarding-table` can incur a race condition with sessions completing and being removed, which could lead to highway crashing and restarting.
```
Comment (hadrielk) [inline docs/release_notes_128t_6.1.md:117]: that was fixed in `6.1.1`, and already noted in that part of this release-notes doc.

File: docs/release_notes_128t_6.1.md [hunk 5]
```diff
@@ -49,6 +49,81 @@ This issue will be corrected in an upcoming release.
...
+------
+- **I95-51093 Segfault in Highway when using multi-threaded ServiceArea:** Resolved an issue when `session-scaling` is set to `enabled` together with `outbound-only`, whereby a race condition can cause a crash and restart of the highway process in the SSR.
```
Comment (hadrielk) [inline docs/release_notes_128t_6.1.md:119]: this was also fixed in `6.1.1`

**Review by hadrielk — CHANGES_REQUESTED** [review]
remove the bugs that were fixed previously

File: docs/release_notes_128t_6.1.md (hunk 4 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_6.1.md]: removed

File: docs/release_notes_128t_6.1.md (hunk 5 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_6.1.md]: removed - thanks for catching those.

**Review by MichaelBaj — DISMISSED** [review]

**Review by hadrielk — DISMISSED** [review]

**Review by MichaelBaj — APPROVED** [review]

**Pull request merged**